lib/migrations/README.md

```diff
@@ -0,0 +1,70 @@
+# Libero Migrations
+
+This package provides an opinionated set of commands that can be used by the reviewer services for database migrations
+
+## Migration CLI script
+
+Its recommended to create a cli script that imports the package.
+
+```js
+import { Cli, Commands } from '@libero/migrations';
+import Config from './config';
+
+const cli = new Cli({
+    banner: 'Libero Reviewer Acme Service: Migration tool',
+    name: 'yarn migrate',
+    knexConfig: Config.knex,
```

Comment:
Could we make some of these arguments the default - i.e. default is by convention ?